.goreleaser.yaml

```diff
@@ -0,0 +1,59 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines bellow are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    # - go mod tidy
+    # you may remove this if you don't need go generate
+    # - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      # - windows
+      - darwin
+      - js
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - wasm
+    
+    ignore: # List of combinations of GOOS + GOARCH + GOARM to ignore.
+      # - goos: windows
+      #   goarch: arm64
+      # - goos: windows
+      #   goarch: arm
+
+    flags:
+    - -tags=b_tiny,b_norepl
```

Comment:
@refaktor, uere are the release build tags defined, including the b_norepl

Reply:
Because omitting `b_norepl` tag is not ok for wasm build I made an override:
- #58